webapp/components/survey/CategoryDetails/store/state/state.js

```diff
@@ -1,66 +1,29 @@
-import * as R from 'ramda'
+export { create } from './create'
 
```

Comment:
can't you export all these from index directly ?

Reply:
uhm, but how can you export all of them as {State} from index?

Reply:
You import your everything and then 
```
export const State = {
getA,
getB,
...
assocA,
..
}
```

Reply:
ok